cmd/agent/dist/conf.d/oracle.d/README.md

```diff
@@ -0,0 +1,2 @@
+
+The configuration directory ``oracle-dbm.d`` is deprecated. Use ``oracle.d`` instead.
```

Comment:
Would it make sense to place this `oracle-dbm.d/README.md` too?

Reply:
Done!